test/plugins/json/test_json.cc

```diff
@@ -30,6 +30,65 @@ JsonDioTestWrapper::GetJsonObject (Json::Value& root, std::string path)
 }
 
 bool
+JsonDioTestWrapper::assert_invalid_context (struct disir_mold *mold, const char *name,
+                                            const char *context_type, const char *errormsg)
+{
+    struct disir_context *context;
+    enum disir_status status;
```

Comment:
nitpick: enum disir_status status has always been the first variable declaration, if present.